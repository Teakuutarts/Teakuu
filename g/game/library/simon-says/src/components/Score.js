import styled from 'styled-components';
import { colors } from '../constants';

const Score = styled.div`
  position: absolute;
  top: 0;
  bottom: 0;
  right: 0;
  left: 0;
  margin: auto;
  background-color: ${colors.dark};
  font-size: ${props => props.length <= 2 ? '62px' : '45px'};
  border-radius: 50%;
  width: 120px;
  height: 120px;
  line-height: 120px;
  text-align: center;
  z-index: 2;
  padding: 5px;
  box-shadow: 0 0 15px rgba(0, 0, 0, .8);
  text-align: center;
`;

export default Score;
