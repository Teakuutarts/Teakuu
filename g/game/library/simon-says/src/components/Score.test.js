import React, { Component } from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Score from './Score';

it('renders correctly', () => {
  const wrapper = shallow(
    <Score />
  );
  expect(toJson(wrapper)).toMatchSnapshot();
});
