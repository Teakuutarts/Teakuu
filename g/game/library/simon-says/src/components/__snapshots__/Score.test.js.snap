exports[`test renders correctly 1`] = `
<div
  className="kxQuRg" />
`;
